tools: update query-status descriptions of dm (#1653)
pingcap/docs: dev/how-to/get-started/data-migration.md

port = 3307
```

> - If you migrate data from MySQL Server, Percona Server, Percona XtraDB Cluster, Amazon Aurora or RDS, set the `flavor` option to `"mysql"`, which is the default value. For this default value, MySQL versions between 5.5 (not included) and 8.0 (not included) are supported.

Reviewer: ```suggestion
- If you migrate data from MySQL Server, Percona Server, Percona XtraDB Cluster, Amazon Aurora or RDS, set the `flavor` option to `"mysql"`, which is the default value. This value is valid only when you are using a MySQL version between 5.5 (not included) and 8.0 (not included).
```
I rewrote the sentence to make it clearer. Is this what you mean? @TomShawn @csuzhangxc

> - If you migrate data from MariaDB Server or MariaDB (Galera) Cluster, set `flavor = "mariadb"`, which only supports MariaDB version later than 10.1.2.

Reviewer: ```suggestion
- If you migrate data from MariaDB Server or MariaDB (Galera) Cluster, set `flavor = "mariadb"`. You can set this value only when you are using a MariaDB version later than 10.1.2.
```
Same as above.

> - Starting from DM 1.0.2, DM automatically generates the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.

Reviewer: ```suggestion
- Starting with DM 1.0.2, DM automatically generates values for the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.
```
Please confirm with the author on "which" DM automatically generates - "the values of the options" or "the options + their values". @TomShawn @csuzhangxc

Author: "the values of the options"
